swervolf.core

```diff
@@ -70,6 +70,10 @@ targets:
     generate : [intercon, swerv_default_config, version]
     parameters : [SIMPRINT=true, jtag_vpi_enable, ram_init_file=hello.vh, rom_init_file, signature, timeout, vcd]
     tools:
+      rivierapro:
+        vlog_options :
```

Comment:
Instead of having to specify compilation_mode on the command-line, it's better to just set `compilation_mode : common` in the rivierapro tool options